gapis/api/vulkan/api/queue.api

```diff
@@ -206,6 +212,7 @@ cmd VkResult vkQueueSubmit(
           signal_semaphores_all_valid.b = false
           vkErrorInvalidSemaphore(ss)
         } else {
+          Semaphores[ss].State = Submitted
```

Comment:
I think this is not perfectly correct, but it might be fine enough.
The issue here is that this will get submitted "before" the execute. I feel like we probably want to mark them as submitted after the execute.

Reply:
After the execution, should not they pass to the "signaled" state?